src/Stripe.net/Constants/SourceType.cs

```diff
@@ -10,8 +10,6 @@ public static class SourceType
 
         public const string Bancontact = "bancontact";
 
-        public const string Bitcoin = "bitcoin";
```

Comment:
Flagging so that it's explicit and we should write down our thoughts here. Basically it means if you look at an old Source for example on an old Charge, then you can not get the source type or its properties.
I still think it's the right choice, but that's what has blocked us for so long.